github-actions/verify-pr-title/test/verifyPrTitle.test.ts

```diff
@@ -0,0 +1,50 @@
+import { expect } from "chai";
```

Comment:
I only tested the regex and not the workflow. Mocking the workflow didn't seem to have the ROI. Let me know if you disagree and I can add tests.